app/models/user.rb

```diff
@@ -1,10 +1,9 @@
 class User < ApplicationRecord
-  include Personable
+  include Personable, Registerable
 
   has_secure_password
 
-  validates :password, presence: true, on: :create
-  validates :password_confirmation, presence: true, on: :create
+  validates :password, length: { minimum: 6 }, allow_nil: true
```

Comment:
Do you want to allow nil password? It feels like we should require it to always be present on User objects. In my mind, it's the definition of a User: it's a person with credentials.

Reply:
password is a virtual attribute and is nil when the user isn't updating their password -- the digest is saved in it's own column